DATA/common/setenv.sh

```diff
@@ -152,6 +152,7 @@ ITSMFT_STROBES=""
 [[ ! -z ${ITS_STROBE:-} ]] && ITSMFT_STROBES+="ITSAlpideParam.roFrameLengthInBC=$ITS_STROBE;"
 [[ ! -z ${MFT_STROBE:-} ]] && ITSMFT_STROBES+="MFTAlpideParam.roFrameLengthInBC=$MFT_STROBE;"
 
+if [[ -z "${SKIP_TPC_CLUSTERSTRACKS_OUTPUT:-}" ]];      then export SKIP_TPC_CLUSTERSTRACKS_OUTPUT=1; fi 
```

Comment:
```suggestion
: ${SKIP_TPC_CLUSTERSTRACKS_OUTPUT:=} 
```
Should it not be set to an empty string by default? That could be achieved the way I wrote it above. But if you can include https://github.com/AliceO2Group/AliceO2/pull/11187 the way I suggested then we would not need this here. Unfortunately I overlooked this variable

Reply:
I don't really know what the default should be :-) I thought it was that we *do* skip the TPC_CLUSTERSTRACKS. 
If you like it more, I can take the suggested changes in O2, and close this one. I don't have a strong opinion.

Thanks for the feedback!!